ci-droid-tasks-consumer-services/src/main/java/com/societegenerale/cidroid/tasks/consumer/services/model/DeleteResourceAction.java

```diff
@@ -0,0 +1,21 @@
+package com.societegenerale.cidroid.tasks.consumer.services.model;
+
+import com.societegenerale.cidroid.api.IssueProvidingContentException;
+import com.societegenerale.cidroid.api.ResourceToUpdate;
+import com.societegenerale.cidroid.api.actionToReplicate.ActionToReplicate;
+import org.apache.commons.lang3.NotImplementedException;
+
+import java.util.Map;
+
+public class DeleteResourceAction implements ActionToReplicate {
```

Comment:
needs to be moved to extensions, so that CI-droid UI is also aware of it and can propose it in UI